### DISCUSSION TITLE
Resize CalendarExt3 month view

### DISCUSSION THREAD

**Discussion by lefebe** [MMRIZE/MMM-CalendarExt3#34] [Q&A]
I am working with a horizontal layout and trying to resize the Ext3 month view so that it is on the right 2/3 of the screen (similar to how the default DAKboard looks).  I have tried editing the Ext3 css, the custom css, and creating unique regions but haven't had much luck.  I am able to make small adjustments but overall haven't had much luck.  Maybe there's an obvious way to do it, but I am still learning my way around MM and Ext3.

I would really appreciate any guidance!

Thank you!

**Comment by eouia** [comment]
Which region did you put the module on?

**Comment by KristjanESPERANTO** [comment]
I am closing the discussion due to lack of feedback. Feel free to reopen it if you continue to have the problem.
